src/Client/QueryFuzzer.cpp

```diff
@@ -860,11 +860,14 @@ void QueryFuzzer::notifyQueryFailed(ASTPtr ast)
         }
     };
 
-    if (const auto * create = ast->as<ASTCreateQuery>())
-        remove_fuzzed_table(create->getTable());
+    if (ast)
```

Comment:
(This and Client.cpp changes are for this fuzzer UBSAN error: https://pastila.nl/?00fa470e/288aebc5582a8f3a05f0468365bf83bb - ast_to_process was null, idk why.)